CMakeLists.txt

```diff
@@ -82,14 +84,32 @@ else()
 endif()
 
 ################################################################################
-configure_file(
-  "${PROJECT_SOURCE_DIR}/include/cnr_param/config.h.in"
-  "${CNR_INSTALL_INCLUDE_DIR}/config.h" 
-  @ONLY)
+#configure_file(
```

Comment:
I would leave it (with a different install directory):
configure_file(
  "${PROJECT_SOURCE_DIR}/include/cnr_param/config.h.in"
  "${CMAKE_CURRENT_BINARY_DIR}/include/cnr_param/config.h" 
  @ONLY)
I such a way, if someone does not want to use Moder CMake and targets can easily understand the compilation process